src/program/db/db_functions.py

```
@@ -1,24 +1,205 @@
+    if filter.year:
+        stmt = stmt.where(MediaItem.year.in_(filter.year))
+    if filter.country:
+        stmt = stmt.where(MediaItem.country == filter.country)
+    if filter.language:
+        stmt = stmt.where(MediaItem.language == filter.language)
+    if filter.requested_by:
+        stmt = stmt.where(MediaItem.requested_by == filter.requested_by)
+
+    if filter.has_file is not None:
+        if filter.has_file:
+            stmt = stmt.where(MediaItem.file != None)
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)